src/core/ext/transport/chttp2/transport/parsing.cc

```diff
@@ -588,7 +588,11 @@ static grpc_error *init_header_frame_parser(grpc_exec_ctx *exec_ctx,
         GRPC_CHTTP2_IF_TRACING(gpr_log(
             GPR_ERROR, "ignoring new grpc_chttp2_stream creation on client"));
       }
-      return init_skip_frame_parser(exec_ctx, t, 1);
+      init_skip_frame_parser(exec_ctx, t, 1);
```

Comment:
I know this unconditionally returns GRPC_ERROR_NONE at this point, but would be good practice to track that and return it, regardless of handling this priority edge cases

Reply:
Good point. I amended.